src/store/index.ts

```diff
@@ -68,11 +66,11 @@ const reducers = combineReducers({
 })
 
 export type AppReduxState = CombinedState<{
-  [PROVIDER_REDUCER_ID]?: ProviderRecord
+  [PROVIDER_REDUCER_ID]?: ProviderState
   [SAFE_REDUCER_ID]: SafeReducerMap
   [NFT_ASSETS_REDUCER_ID]?: NFTAssets
   [NFT_TOKENS_REDUCER_ID]?: NFTTokens
-  [TOKEN_REDUCER_ID]?: Map<string, Token>
+  [TOKEN_REDUCER_ID]?: TokenState
```

Comment:
same here

Reply:
the moment I created my version of the AppReduxState, the app failed to build due to bad typing in test files (:question:). There was a test that built the store with only one reducer or something about that.

I've just removed and now it's not failing :face_with_thermometer: